styles.css

```diff
@@ -343,3 +346,276 @@ footer {
 .footerHr {
   color: #dfe1e6;
 }
+
+@media screen and (max-width: 767px) {
+  .noMobile { display: none; }
+}
+
+@media screen and (min-width: 768px) {
+  #headline {
+    display: flex;
+    height: 798px;
+    flex-direction: column;
+    padding-top: 192px;
+    padding-bottom: 251px;
+    padding-left: 2.5vw;
+    background: #1c1a19 url("img/desktopIntroBkg.png") no-repeat;
+    background-position: 100% 0;
+  }
+
+  .headlineContent {
+    display: flex;
+    flex-direction: row-reverse;
+    width: 39.3vw;
+    gap: 6vw;
+  }
+
+  .introText {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .introText > h1 {
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .introText > .lh32 {
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .introText > .lh24 {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .socialIcons > ul {
+    padding-top: 87px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+
+  #works {
+    display: flex;
+    padding: 135px 9.9vw 130px 9.7vw;
+    height: auto;
+  }
```

Comment:
- [ ] On the works section, remember to follow the margin and padding on each element (you can check them on the template on FIGMA) so it has the exact space between each element, in this way the design will look better and have more attraction on the view of the visitor


![image](https://user-images.githubusercontent.com/93630700/206734822-6635abe2-4a69-49e4-a803-b1f6d2f8abd2.png)